src/Zoo.java

```diff
@@ -7,10 +7,11 @@
    *
    * @param list list of birds of particular type
    */
-  public static void feed(List list) {
-    // TODO fix method declaration
-    System.out.println("Feeding birds");
-    // TODO print birds in the cycle
+  public static void feed (List<? extends Bird> list) {
+
+    for (Bird bird : list) {
+      System.out.println("Feeding birds: " + bird.getClass().getSimpleName());
+    }
```

Comment:
Looks like you forked repo a little bit early, since there is a method `eat` in the `Bird` class:
```
  public void eat() {
    // TODO print a message that bird is eating
  }
```